apis/r/src/rinterface.cpp

```diff
@@ -585,6 +585,50 @@ const char *_tiledb_filter_to_string(tiledb_filter_type_t filter) {
     }
 }
 
+// internal helper function
+// filter options taken from tiledb-r
+// https://github.com/TileDB-Inc/TileDB-R/blob/525bdfc0f34aadb74a312a5d8428bd07819a8f83/src/libtiledb.cpp#L369-L388
+Rcpp::List _get_filter_opts(Rcpp::XPtr<tiledb::Filter> filter) {
+    auto filter_type = _tiledb_filter_to_string(filter->filter_type());
+
+    // catches are intentionally blank to leave the values as NA
+    int compression = R_NaInt, bytewidth = R_NaInt, factor = R_NaInt, offset = R_NaInt;
+    uint32_t bit_width = R_NaInt, positive_delta = R_NaInt;
```

Comment:
Let's check with the core folks and see if there's a more elegant way to do this.

I know you're copying from existing code in TileDB-R, so, obviously leaving this way isn't the worst thing -- however, a catch-anything can mask other unexpected errors. If we can find a more elegant way, I'd love for us to do so. cc @teo-tsirpanis .

Reply:
This function is also implemented in #3851, and so whatever resolution we adopt for https://github.com/single-cell-data/TileDB-SOMA/pull/3851#discussion_r2014973214 will end up being used here

Reply:
Addresed in #3851